src/routes/safe/components/TransactionsNew/index.tsx

```diff
@@ -0,0 +1,50 @@
+import React from 'react'
+import { useSelector, useDispatch } from 'react-redux'
+import { currentPageSelector, newTransactionsCurrentPageSelector } from '../../store/selectors/newTransactions'
+import { useFetchNewTransactions } from '../../container/hooks/useFetchNewTransactions'
+import { ButtonLink, Loader } from '@gnosis.pm/safe-react-components'
+import { nextPage, previousPage } from '../../store/actions/transactionsNew/pagination'
+import { Transaction } from '../../store/models/types/transactions'
+
+const Transactions = (): React.ReactElement => {
+  const dispatch = useDispatch()
+  const transactions = useSelector(newTransactionsCurrentPageSelector)
+  const { currentPage, maxPages } = useSelector(currentPageSelector)
+  useFetchNewTransactions()
+
+  // TODO: Remove this once we implement infinite scroll
+  const nextPageButtonHandler = () => {
+    dispatch(nextPage())
+  }
+
+  const previousPageButtonHandler = () => {
+    dispatch(previousPage())
+  }
+
+  if (!transactions) return <div>No txs available for safe</div>
+
+  if (!transactions.length) return <Loader size="lg" />
```

Comment:
this way is a little bit weird to check request state. we can make use of this enum and store it in redux

```
export enum requestStates {
  notAsked = "notAsked",
  loading = "loading",
  loaded = "loaded",
  failed = "failed",
}
```

Reply:
I would propose to modify it in another pr, because it will add more logic that we don't need for now.